Update x12-edi-connector.adoc (#754)
mulesoft/docs-connectors: modules/ROOT/pages/x12-edi/x12-edi-connector.adoc

> The general tab lets you configure settings for reading and writing X12 messages:

Reviewer: ```suggestion
The general tab enables you to configure settings for reading and writing X12 messages:
```
...and then reword the following list items so that they are PARALLEL: all settings (noun-based) or all user actions (verb-based).
Edits to existing list items are for reference only; I'd like to see the list once more after you make it parallel.
If you don't have time for that, then the edits will at least be a step in the right direction!  :-)

Reviewer: GLOBALLY remove periods at the ends of list fragments and move subsequent sentence or sentences to a new para below the list item, as shown in the green edit below:

* X12 form and validation (standard X12, HIPAA SNIP Type 1, or HIPAA SNIP Type 2).

> * Manually create or edit the list of schema definitions for the message structures to use. They must all use the same form (either X12 or HIPAA). **Note:** To use the Functional Acknowledgment schema type (997 or 999) it must match the **Generate 999 Functional Acknowledgments** setting on the Parser tab (`false` for 997, `true` for 999).

Reviewer: ```suggestion
* Manually create or edit the list of schema definitions for the message structures to use. +
The schema definitions must all use the same form: either X12 or HIPAA. +
To use Functional Acknowledgment schema type 997 or 999, the definition must match the *Generate 999 Functional Acknowledgments* setting on the *Parser tab* (`false` for 997, `true` for 999).
```
Is the Functional Acknowledgement type part of the list of schema definitions, or is this something separate that should perhaps be its own list item?

* Generate 999 Implementation Acknowledgments, rather than the 997 Functional Acknowledgments generated by default. Note that the support for 999 Implementation Acknowledgments does not include CTX segment generation.

> * Report segment error details to sender in the generated 997/999 acknowledgments.

Reviewer: ```suggestion
* Report segment error details to sender in the generated 997 or 999 acknowledgments.
```

* Automatically include a standard 997 or 999 schema that matches the generated acknowledgment in the configuration for received messages.

> * Set the path to a Functional Acknowledgment schema to be used for generated acknowledgments such as `/hipaa/005010X231A1/999.esl` which uses the standard HIPAA 005010X231A1 acknowledgment.

Reviewer: ```suggestion
* Set the path to a Functional Acknowledgment schema to be used for generated acknowledgments such as `/hipaa/005010X231A1/999.esl`, which uses the standard HIPAA 005010X231A1 acknowledgment. +
```
This is not part of the green, changed text, but I wonder if there should be mention made in the first list about 997 being the default between 997 and 999?

>  Using this means the schema type (997 or 999) must match the Generate 999 Functional Acknowledgments setting (false for 997, true for 999).

Reviewer: ```suggestion
The schema type included in the path must match the Generate 999 Functional Acknowledgments setting (`false` for 997, `true` for 999).
```

>  If not set, the parser uses a hardcoded X12 997 or 999 schema, as appropriate.

Reviewer: ```suggestion
 If you do not set the path, the parser uses a hardcoded X12 997 or 999 schema, as appropriate.
```
Is this a problem?

> 

Reviewer: I looked at the output for this file and noticed several issues with title capitalization and list-item format.
So...this whole thing could use a copyedit at some point.
Does this deliverable follow the connector template?
